test/Fail/AnonymousRecConstrAmbig.err

```diff
@@ -0,0 +1,4 @@
+AnonymousRecConstrAmbig.agda:9,5-19: error: [ConstructorNameOfNonRecord]
+Only record types have constructor names, but mk is the name of an
+ambiguous constructor.
+when scope checking mk.constructor
```

Comment:
This error is a bit surprising.
I would have expected a complained that `mk` is not a record module.
Why is `.constructor` a method of a type, not of a record module?

I am also a bit confused by the dual role of "constructor" in this error message.
And why does it matter that `mk` is an _ambiguous_ constructor?

Maybe "record type" is more understandable for the general audience.
But maybe in a formulation like: "Component `.constructor` can only be taken from a record type, but `mk` isn't one". 

Dunno.

Reply:
>  I would have expected a complained that `mk` is not a record module.
Why is `.constructor` a method of a type, not of a record module?

Well, because the record module is parametrized by an instance of the record, so the constructor doesn't really behave like it's a member of that module, even though the syntax is pretending like it is. So I thought "record type" was more appropriate.

Reply:
Maybe just dropping the "ambiguous" from this error message is sufficient.